### PR TITLE
Copy Google AI Studio templates from docs, and update.

### PR DESCRIPTION
**Pull request by MarkDaoust**
copied from: google/generative-ai-docs@98b2114d01e839e81cc685aa42f8f80c637138c8

### CODE REVIEW

File: templates/aistudio_gemini_prompt_freeform.ipynb [hunk 1]
```diff
@@ -0,0 +1,474 @@
+{
```
Comment (markmcd) [inline templates/aistudio_gemini_prompt_freeform.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    gais_contents=[
</pre><p>Is this testing data? or part of the template?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (markmcd) [inline templates/aistudio_gemini_prompt_freeform.ipynb:1]: <p>Doesn't have to be part of this change but WDYT about explaining why we need this section?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (markmcd) [inline templates/aistudio_gemini_prompt_freeform.ipynb:1]: <p>Do we need this if the previous cell is outputting <code>candidates</code> already? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Review by markmcd — DISMISSED** [review]
Could you run `pyink --pyink-indentation=2` on this too? It'll tidy up some of the whitespace.

**Comment by markmcd** [comment]
oh this is a draft. woops sorry I didn't notice. :facepalm:

File: templates/aistudio_gemini_prompt_freeform.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply templates/aistudio_gemini_prompt_freeform.ipynb:1]: Fixed.

Comment (MarkDaoust) [reply templates/aistudio_gemini_prompt_freeform.ipynb:1]: <p>Done.</p>

Comment (MarkDaoust) [reply templates/aistudio_gemini_prompt_freeform.ipynb:1]: <p>Done.</p>

**Review by markmcd — APPROVED** [review]

**Pull request merged**